### DISCUSSION TITLE
Minecraft won't luanch with my modset?

### DISCUSSION THREAD

**Discussion by C00lerDude** [FabricMC/fabric-api#3400] [FAQ] [answered]
Why won't my minecraft luanch, here is crash report plus my list on mods:

Crash report: [crash-2023-11-01_14.57.39-client.txt](https://github.com/FabricMC/fabric/files/13231257/crash-2023-11-01_14.57.39-client.txt)

Mods: 
![Screenshot 2023-11-01 150601](https://github.com/FabricMC/fabric/assets/135157839/28b75dad-985a-44d5-a9c3-0a8d551dba5c)

Thanks!

**Comment by Linguardium** [comment]
VMP is including a 1.20.1 pre-release mod in their 1.20.2 alpha releases. remove vmp

**Reply by C00lerDude** [reply]
I tried that and got the same error code and crash report. Any other Solutions?

**Reply by Linguardium** [reply]
post the new "same" crash report then

**Reply by Linguardium** [reply]
i would say to update memoryleakfix if you havent already, as well

**Reply by C00lerDude** [reply]
Here is the new crash report: [crash-2023-11-01_16.26.40-client.txt](https://github.com/FabricMC/fabric/files/13232007/crash-2023-11-01_16.26.40-client.txt)
and i have updated MemoryLeakFix already.

**Reply by Linguardium** [reply, marked as answer]
Caused by: org.spongepowered.asm.mixin.throwables.MixinApplyError: Mixin [memoryleakfix-fabric.mixins.json:customPayloadLeak.ClientboundCustomPayloadPacket_bufferMixin from mod memoryleakfix] from phase [DEFAULT] in config [memoryleakfix-fabric.mixins.json] FAILED during APPLY

Caused by: org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException: @Shadow field field_12162 was not located in the target class 

definitely appears to be a memoryleakfix issue

**Reply by C00lerDude** [reply]
Hmmm okay, let me get rid of it and try it agian!

**Reply by C00lerDude** [reply]
It worked! Thank you soo much!
